src/exchange_asset.rs

```diff
@@ -193,13 +194,27 @@ pub fn try_complete_asset_exchange(
     let total_capital: i64 = exchanges.iter().filter_map(|e| e.capital).sum();
     let abs_capital = total_capital.unsigned_abs();
     if total_capital > 0 {
-        let send_capital = BankMsg::Send {
-            to_address: to.unwrap_or(info.sender).into_string(),
-            amount: coins(abs_capital.into(), state.capital_denom),
+        match state.fiat_deposit_addr {
+            Some(fiat_deposit_addr) => {
+                let fiat_deposit_transfer = wasm_execute(
+                    fiat_deposit_addr.into_string(),
+                    &FiatDepositExecuteMsg::Transfer {
+                        amount: abs_capital.into(),
+                        recipient: to.unwrap_or(info.sender).into(),
+                    },
+                    vec![],
+                )?;
+                response = response.add_message(fiat_deposit_transfer)
+            }
+            None => {
+                let send_capital = BankMsg::Send {
+                    to_address: to.unwrap_or(info.sender).into_string(),
+                    amount: coins(abs_capital.into(), state.capital_denom),
+                };
+                response = response.add_message(send_capital)
+            }
```

Comment:
I just realized we could avoid a mut response var if we wanted by redeclaring in places like line 196, and have the conditionals return a new or the existing response val